_posts/2020/2020-08-20-pmdk-pvs-studio.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Static code analysis of the PMDK
+author: michalbiesek
+layout: post
+identifier: pmdk-static-analysis
+---
+
+# Introduction
+
+In the PMDK team, we emphasize the quality of the code. The high quality and failure-free runtime are especially crucial in memory management software. Since there is no silver bullet for avoiding bugs, we already use two different static analysis tools and many runtime checkers e.g. [valgrind]. One of the standard practices in the software development is a static code analysis, which improves the overall project quality and fixes bugs in the early stage of development. Improving static analysis effectiveness is a separate academic problem. What is worth mentioning is the fact that different tools take a various approach. With combined advantages of Open Source projects such as transparency and reliability, as well as a new static analysis tool, we get a new bag of tricks and a fresh look to an old problem :).
```

Comment:
Break this up into multiple lines.

we focus on the quality of our codebase. strike the second sentence. Move the "one of the standard ..." sentence before the "Since there is no silver bullet" sentence.

Reply:
Done.